src/core/core.scale.js

```diff
@@ -100,6 +102,23 @@ function computeTextSize(context, tick, font) {
 }
 
 module.exports = Element.extend({
+	/**
+	 * @private
+	 */
+	_configure: function() {
```

Comment:
I think this is an interesting concept for supporting these options

Reply:
Only things that should need an `update` to change can be resolved here though.